src/container/interfaces.ts

```diff
@@ -47,3 +74,47 @@ export interface ContainerProps extends BaseComponentProps {
    */
   variant?: 'default' | 'stacked';
 }
+
+export namespace ContainerProps {
+  export interface Media {
+    /**
+     *
+     * Use this slot to render your media element. We support `img`, `video` and `picture` elements.
+     *
+     * The value for each media property can either be a single value (which applies for all breakpoints) or an object where the key
+     * is one of the supported breakpoints (`xxs`, `xs`, `s`, `m`, `l`, `xl`) or `default`.
+     * The value of this key is applied for that breakpoint and those above it. You must provide a `default` value for `content`.
+     */
+    content: React.ReactNode | MediaDefinition.ContentBreakpointMapping;
+
+    /**
+     * Defines the media slot's position within the container. Defaults to `top`.
+     */
+    position?: 'top' | 'side' | MediaDefinition.BreakpointMapping<'top' | 'side'>;
+
+    /**
+     * Defines the width of the media slot when positioned on the side. Corresponds to the `width` CSS-property.
+     * When this value is set, the media element may be cropped, with `object-fit: cover` centering it.
```

Comment:
When this value is set, media elements larger than the defined width may be cropped, with 'object-fit: cover' centering it.